drivers/intellicache-clean

```diff
@@ -21,7 +21,7 @@ import sys
 import XenAPIPlugin
 sys.path.append("/opt/xensource/sm/")
```

Comment:
Path override probably no longer required?

Reply:
Do though, for the same reason as above. Can drop these when everything that gets imported no longer references anything in the compat path.

(this is why ```/usr/libexec/sm/mpathcount``` _doesn't_ need it)